### PR TITLE
Feat: linking endpoint overlay to endpoints

### LINKED ISSUE #502: Link hosts to endpoint overlay

### PR DESCRIPTION
**Pull request by stephanwampouille**
dependent on: #505 ,
closes #502 

Don't mind the moving endpoints, thats just how Francis set up his mock data
https://user-images.githubusercontent.com/84744117/216452589-bddc94c2-da1e-48e5-852b-896d97501591.mov

### CODE REVIEW

File: packages/ds/src/components/tooltip/tooltip.tsx [hunk 1]
```diff
@@ -11,12 +11,23 @@ export const Tooltip: React.FC<{
   show?: boolean;
 }> = ({ message, children, position = "top", show = true }) => {
   const [hovered, setHovered] = useState(false);
+  const [hoverAttention, setHoverAttention] = useState(false);
+
+  useEffect(() => {
```
Comment (kenrick) [inline packages/ds/src/components/tooltip/tooltip.tsx:16]: we should make this an option on tooltip. Like `<Tooltip defaultHover=200....>`

Comment (stephanwampouille) [reply packages/ds/src/components/tooltip/tooltip.tsx:16]: So would you like me to revert this component but add an option for a delayed hover like mine was or just have them input the delay number? for example `defaultHover=0` would be like how it was before.

Comment (stephanwampouille) [reply packages/ds/src/components/tooltip/tooltip.tsx:23]: lmk if you like what I did

File: packages/ds/src/components/tooltip/tooltip.tsx [hunk 2]
```diff
@@ -9,14 +9,32 @@ export const Tooltip: React.FC<{
   children: React.ReactNode;
   position?: "top" | "bottom";
   show?: boolean;
-}> = ({ message, children, position = "top", show = true }) => {
+  defaultHover: number;
```
Comment (stephanwampouille) [inline packages/ds/src/components/tooltip/tooltip.tsx:12]:
```suggestion
  defaultHover?: number;
```

**Pull request merged**